scripts/verify.sh

```diff
@@ -30,7 +30,7 @@ verify_if_multiple_pods_exists() {
     echo "Verifying if multiple pods exists"
     count=0
     MULTIPLE_POD_EXISTS=false
-    while [ "$count" -le 3 ]; do
+    while [ "$count" -le 10 ]; do
```

Comment:
@cxznmhdcxz Please add the reason why we are increasing the count from 3 to 10 here. Thanks.